chatbot/db/db.js

```diff
@@ -5,18 +5,23 @@ const { Pool } = require('pg')
 require('dotenv').config();
 
 const pool = new Pool({
-    host: process.env.PG_HOST,
-    port: process.env.PG_PORT,
-    user: process.env.PG_USER,
-    database: process.env.PG_USER,
-    password: process.env.PG_PASSWORD,
+    host: getEnvVar('PG_HOST'),
+    port: getEnvVar('PG_PORT'),
+    user: getEnvVar('PG_USER'),
+    database: getEnvVar('PG_USER'),
+    password: getEnvVar('PG_PASSWORD'),
     ssl: true
 })
 
 pool.on('error', (err) => {
     console.error('unexpected database error:', err)
 })
 
+function getEnvVar(name) {
```

Comment:
This is just copy-and-pasted from `server.js`, right?

Instead, you could create new `helpers.js` file with this function in it and the use it in both `server.js` and here in `db.js`. That way if we ever need to change it for some reason, it'll work in both places.